sponsorship_compassion/model/contracts.py

```diff
@@ -61,8 +67,43 @@ def _get_contract_from_invoice(invoice_obj, cr, uid, invoice_ids,
                         contract.write({
                             'activation_date': datetime.today().strftime(DF)})
 
+                        # Cancel the invoices if a contract is activated

```

Comment:
Please change comment and name method to "cancel_old_invoices", as it is strange to read "Cancel invoices when sponsorship is activated." We don't wan't to cancel all invoices, but only those that are old.